docs/learn-accounts.md

```diff
@@ -353,3 +353,17 @@ need to place the deposit and Alice will receive her deposit back.
 - [Understanding Accounts and Keys in Polkadot](https://www.crowdcast.io/e/polkadot-keys) - An
   explanation of what the different kinds of accounts and keys are used for in Polkadot, with Bill
   Laboon and Chinmay Patel of BlockX Labs.
+
```

Comment:
I would recommend just placing this above Resources - as I think that should be the last section of this page